tests/js/spec/views/projectsDashboard/index.spec.jsx

```diff
@@ -47,150 +53,241 @@ describe('ProjectsDashboard', function () {
   });
 
   describe('empty state', function () {
-    it('renders with no projects', function () {
-      const noProjectTeams = [TestStubs.Team({projects: []})];
+    it('renders with no projects', async function () {
+      const noProjectTeams = [TestStubs.Team({isMember: false, projects: []})];
 
-      const wrapper = mountWithTheme(
+      render(
         <Dashboard
           teams={noProjectTeams}
           organization={org}
           params={{orgId: org.slug}}
         />,
-        routerContext
+        {routerContext}
       );
 
-      expect(wrapper.find('Button[data-test-id="create-project"]').exists()).toBe(false);
-      expect(wrapper.find('NoProjectMessage').exists()).toBe(true);
```

Comment:
would like to check for something to exist, checking 2 things don't exist could potentially not test anything